### DISCUSSION TITLE
Utilização do Metodo de estudos do Curso

### DISCUSSION THREAD

**Discussion by cmsouzaac** [balta-io/2801#112] [Perguntas e Respostas] [answered]
Professor, @andrebaltieri!
Até que ponto ou em qual momento temos de estudar os conteúdos da documentação, tendo em vista que o senhor no curso ja traz o material bem complementar sobre cada assunto. Entendo que a documentação é importante para complemento, mas no primeiro momento devo focar na documentação ou somente para o futuro proximo. Levando em consideração ao material de estudo em ppt. que o senhor ja disponibiliza.

Obrigado.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @cmsoouza, como vai?

Os cursos funcionam como um guia, pra te mostrar novas tecnologias, conceitos e técnicas. Você sempre vai precisar recorrer a livros e documentações para mais detalhes.

Por hora, foca na carreira .NET, se precisar, consulta as docs...
